pymemcache/client/base.py

```diff
@@ -38,6 +39,10 @@
     b'cas':     (b'STORED', b'EXISTS', b'NOT_FOUND'),
 }
 
+SOCKET_KEEPALIVE_SUPPORTED_SYSTEM = [
+    'Linux',
+]
```

Comment:
Let's use a `set` here:

```suggestion
SOCKET_KEEPALIVE_SUPPORTED_SYSTEM = {
    'Linux',
}
```